frontend/packages/erd-core/src/components/ERDRenderer/LeftPane/LeftPane.tsx

```diff
@@ -6,28 +6,17 @@ import {
   SidebarGroupContent,
   SidebarGroupLabel,
   SidebarMenu,
-  SidebarMenuButton,
-  SidebarMenuItem,
   SidebarRail,
 } from '@liam-hq/ui'
-import { Table2 } from '@liam-hq/ui'
-import {
-  updateActiveTableName,
-  useDBStructureStore,
-  useUserEditingStore,
-} from '../../../stores'
-import styles from './LeftPane.module.css'
+import { useNodes } from '@xyflow/react'
+import { useMemo } from 'react'
+import { isTableNode } from '../ERDContent'
 import { TableCounter } from './TableCounter'
-
-const handleClickMenuButton = (tableId: string) => () => {
-  updateActiveTableName(tableId)
-}
+import { TableNameMenuButton } from './TableNameMenuButton'
 
 export const LeftPane = () => {
-  const { tables } = useDBStructureStore()
-  const {
-    active: { tableName },
-  } = useUserEditingStore()
+  const nodes = useNodes()
+  const tableNodes = useMemo(() => nodes.filter(isTableNode), [nodes])
```

Comment:
The `hidden` data is contained in the node information, but it can only be retrieved from useNodes.

ref: https://reactflow.dev/api-reference/types/node#hidden

Therefore, the data displayed in the LeftPane has been changed via `useNodes()`.